Paper-MojangAPI/src/main/java/com/destroystokyo/paper/event/brigadier/CommandRegisteredEvent.java

```diff
@@ -21,6 +21,7 @@
  *
  * @deprecated Draft API - Subject to change until confirmed solves desired use cases
  */
+@Deprecated
```

Comment:
Annotating events with the `@Deprecated` annotation results in the following warning when registering such an event listener: 
```
"%s" has registered a listener for %s on method "%s", but the event is Deprecated. "%s"; please notify the authors %s.
```

Should we maybe annotate these events with the `@Warning(false)` annotation as well, since they are considered Draft API?

Reply:
Good catch